packages/sfui/frameworks/react/components/VsfAccordionItem/VsfAccordionItem.tsx

```diff
@@ -16,10 +18,13 @@ export default function VsfAccordionItem({
   const classes = classNames([className, 'vsf-accordion-item']);
   return (
     <details className={classes} open={open} onToggle={onToggle} {...attributes} data-testid="accordion-item">
-      <summary className={`${chevronLeft && 'vsf-accordion-item__header--right'} vsf-accordion-item__header`}>
+      <summary className={classNames('vsf-accordion-item__header', chevronLeft && 'vsf-accordion-item__header--right')}>
```

Comment:
Last one, and we're good to go. 😄 
```suggestion
      <summary className={classNames('vsf-accordion-item__header', { 'vsf-accordion-item__header--right': chevronLeft })}>
```